app/models/conditional.rb

```diff
@@ -16,6 +16,7 @@ def initialize(attributes)
 
   def to_metadata
     {
+      '_uuid' => generate_uuid,
```

Comment:
now that i think about it, will this not generate a different uuid every time to_metadata is called?